ihmc-robotics-toolkit/src/test/java/us/ihmc/robotics/math/frames/YoMatrixTest.java

```diff
@@ -65,6 +67,8 @@ public void testYoMatrixDimensioning()
       {
       }
 
+      yoMatrix.reshape(maxNumberOfRows, maxNumberOfColumns);
```

Comment:
Same here, `yoMatrix.reshape(numberOfRows, numberOfColumns);` is not needed for the test to pass.